tests/test_codecs.py

```diff
@@ -319,3 +324,37 @@ def test_debug_id():
     tags = [t for t in span.tags if t.key == debug_header]
     assert len(tags) == 1
     assert tags[0].value == 'Coraline'
+
+
+def test_non_unicode_baggage(httpserver):
+    # httpserver is provided by pytest-localserver
+    httpserver.serve_content(content='Hello', code=200, headers=None)
+
+    tracer = Tracer(
+        service_name='test',
+        reporter=InMemoryReporter(),
+        # don't sample to avoid logging baggage to the span
+        sampler=ConstSampler(False),
+    )
+    tracer.codecs[Format.TEXT_MAP] = TextCodec(url_encoding=True)
+
+    baggage = [
+        (b'key', b'value'),
+        (u'key', b'value'),
+        (b'key', bytes(chr(255))),
+        (u'caf\xe9', 'caf\xc3\xa9'),
```

Comment:
The test name ` test_non_unicode_baggage` makes it seem like this tests string/byte baggage; but I don't see any assertions on the non unicode baggage items.

Reply:
The potential bug is in the httplib/urllib2 so IDK if there is a direct assertion we should do.